src/components/guide-list.js

```diff
@@ -32,31 +32,36 @@ export const GuideList = ({ subCategoryTitle, guides }) => {
 
   const orderedGuides = orderBy(guides, guide => guide.frontmatter.title);
 
-  const guideList = orderedGuides.map(guide => {
+  const guideList = orderedGuides.map((guide, index) => {
     const { title, slug, description } = guide.frontmatter;
     const guideTitleId = getGuideTitleId(slug);
     const guideDescriptionId = getGuideDescriptionId(slug);
 
     return (
-      <ListItem className={classes.guideInfo} key={title} disableGutters>
-        <IconButton
-          component={Link}
-          className={classes.viewButton}
-          to={`/${slug}`}
-          role="link"
-          aria-labelledby={`${guideTitleId} ${guideDescriptionId}`}
-        >
-          <VisibilityIcon />
-        </IconButton>
-        <ListItemText>
-          <Typography variant="h5" component="p" id={guideTitleId}>
-            {title}
-          </Typography>
-          <Typography variant="body2" id={guideDescriptionId}>
-            {description}
-          </Typography>
-        </ListItemText>
-      </ListItem>
+      <React.Fragment key={title}>
+        <ListItem className={classes.guideInfo} disableGutters>
+          <IconButton
+            component={Link}
+            className={classes.viewButton}
+            to={`/${slug}`}
+            role="link"
+            aria-labelledby={`${guideTitleId} ${guideDescriptionId}`}
+          >
+            <VisibilityIcon />
+          </IconButton>
+          <ListItemText>
+            <Typography variant="h5" component="p" id={guideTitleId}>
+              {title}
+            </Typography>
+            <Typography variant="body2" id={guideDescriptionId}>
+              {description}
+            </Typography>
+          </ListItemText>
+        </ListItem>
+        {index < orderedGuides.length - 1 && (
+          <Divider/>
```

Comment:
A bottom border might have previously been used to provide a visual affect without impacting screen readers.

Let's add `aria-hidden="true"` to be safe:
```suggestion
          <Divider aria-hidden="true" />
```

Aria rules are used a bit different than regular react props.  [Related docs can be found here](https://reactjs.org/docs/accessibility.html).